tests/test_explorer.py

```diff
@@ -0,0 +1,169 @@
+"""Unit tests for explorer module."""
+
+import unittest
+from unittest import mock
+from importlib import reload
+
+from PyQt5.QtCore import QObject, Qt
+from PyQt5.QtWidgets import QApplication, QTreeWidgetItem
+from PyQt5.QtTest import QTest
+
+from iquip.apps import explorer
+
+class ExplorerFrameTest(unittest.TestCase):
+    """Unit tests for ExplorerFrame class."""
+
+    def setUp(self):
+        self.qapp = QApplication([])
+        explorer._FileFinderThread = mock.MagicMock()
+
+    def tearDown(self):
+        del self.qapp
+        reload(explorer)  # Restore explorer module.
+
+    def test_file_tree_item_expanded(self):
+        explorer.ExplorerApp.lazyLoadFile = mock.MagicMock()
+        app = explorer.ExplorerApp(name="name", masterPath="masterPath", parent=QObject())
+        directoryItem = QTreeWidgetItem(app.explorerFrame.fileTree)
+        directoryItem.setText(0, "directory")
+        QTreeWidgetItem(directoryItem)  # Add an empty item to an unloaded directory.
+        directoryItem.setExpanded(True)
+        app.lazyLoadFile.assert_called_once()  # pylint: disable=no-member
```

Comment:
Umm.. You are right. Then, how about including tests for the frame in the app?
The frame is independent to the app, but in order to run well, the app is necessary.
So I want to move several tests such as expanding a directory into the app test.

Please leave your opinion!

Reply:
But the problem is that tests like `test_file_tree_item_expanded()` are not _unit tests_ in our sense, since unit tests should only depend on the target function, method, class, etc.

We can call those tests _functional tests_ and it seems quite important in a sense that we should verify the API specification.

I am not quite sure yet, whether to implement the functional tests along with the unit tests or to separate them.

Reply:
Ah, I understood what you said. I agree with you and can I remove all tests for `ExplorerFrame`?

Reply:
Okay, then let's remove them in this PR, and we can talk about the functional tests later.